### ISSUE TITLE
Enumeration Options

### ISSUE THREAD

**Issue by rahabash** [MartinKuschnik/WmiLight#2]
Does this library limit enumeration options such as EnumerateDeep, Rewindable, Timeout, etc?

**Comment by MartinKuschnik** [comment]
Currently there is no such feature.

**Comment by MartinKuschnik** [comment]
Is there any need for that?

**Comment by MartinKuschnik** [comment]
Now it's implemented in [v2.1.17172.10](https://github.com/MartinKuschnik/WmiLight/releases/tag/v2.1.17172.10).

Usage:
```c#
var opt = new WmiConnectionOptions() { EnablePackageEncryption = true };
var cred = new NetworkCredential("USERNAME", "PASSWORD", "DOMAIN");

using (WmiConnection con = new WmiConnection(@"\\MACHINENAME\root\cimv2", cred, opt))
{
    foreach (WmiObject partition in con.CreateQuery("SELECT * FROM Win32_DiskPartition", EnumeratorBehaviorOption.DirectRead | EnumeratorBehaviorOption.ReturnImmediately))
    {
        Console.WriteLine(partition["Name"]);
    }
}
```

**Comment by rahabash** [comment]
Great, thank you!
